src/main/java/pl/javastart/task/CoordinateSystem.java

```diff
@@ -6,6 +6,36 @@ public class CoordinateSystem {
 
     // uzupełnij metodę. Używaj scannera przekazanego w parametrze.
     void run(Scanner scanner) {
-        System.out.println("Punkt (5, -3)");
+        System.out.println("Podaj wspolrzedna x:");
+        int x = scanner.nextInt();
+        System.out.println("Podaj wspolrzedna y:");
+        int y = scanner.nextInt();
+        System.out.println("Punkt (" + x + ", " + y + ")" + checkQuarter(x, y));
+
+    }
+
+    private String checkQuarter(int x, int y) {
+        if (x > 0 && y > 0) {
+            return " lezy w I cwiartce ukladu wspolrzednych.";
+        }
+        if (x < 0 && y > 0) {
+            return " lezy w II cwiartce ukladu wspolrzednych.";
+        }
+        if (x < 0 && y < 0) {
+            return " lezy w III cwiartce ukladu wspolrzednych.";
+        }
+        if (x > 0 && y < 0) {
+            return " lezy w IV cwiartce ukladu wspolrzednych.";
+        }
+        if (x == 0 && y == 0) {
+            return " lezy w srodku ukladu wspolrzednych.";
+        }
+        if (x != 0 && y == 0) {
+            return " lezy na osi X ukladu wspolrzednych.";
+        }
+        if (x == 0 && y != 0) {
+            return " lezy na osi Y ukladu wspolrzednych.";
+        }
+        return null;
```

Comment:
Tutaj możesz zapisać if-else na zasadzie
```
if (x==0) {
...
}
else {
 ...
}
```
Dzięki temu na końcu nie będzie potrzeby zapisu return null, a z wcześniejszych warunków, w których sprawdziłaś wszystkie kombinacje wynika to, że jedyne pozostałe opcje, to albo oś X, albo oś Y